Assets/[SCRIPTS]/DamageSystem.cs

```diff
@@ -0,0 +1,24 @@
+using UnityEngine;
+
+public class DamageSystem : MonoBehaviour
+{
+    [SerializeField] private int life2 = default;
+    private void OnTriggerEnter2D(Collider2D col)
+    {
+        if (col.CompareTag("smallFist"))
+        {
+            life2 --;
+        }
+        if (col.CompareTag("mediumFist"))
+        {
+            life2--;
+            life2--;
+        }
+        if (col.CompareTag("bigFist"))
+        {
+            life2--;
+            life2--;
+            life2--;
```

Comment:
Create a method to remove X amount of lives, also why is named life2?

Reply:
Because it's the life of the player2 you are resting, maybe a name change will be in order